app/javascript/portal/components/TableOfContents.vue

```diff
@@ -1,20 +1,29 @@
 <template>
-  <div class="hidden lg:block flex-1 scroll-mt-24 pl-4">
-    <div v-if="rows.length > 0" class="sticky top-24 py-12 overflow-auto">
+  <div class="hidden lg:block flex-1 py-6 scroll-mt-24 pl-4">
+    <div v-if="rows.length > 0" class="sticky top-24 py-2 overflow-auto">
       <nav class="max-w-2xl">
         <h2
           id="on-this-page-title"
-          class="text-slate-800 dark:text-slate-50 font-semibold tracking-wide border-b mb-3 leading-7"
+          class="text-slate-800 pl-6 dark:text-slate-50 font-semibold tracking-wide py-3 leading-7 border-l-2 border-solid border-slate-100 dark:border-slate-800"
+          :class="{
+            '!border-slate-400 dark:!border-slate-100': !currentSlug,
```

Comment:
Lets not use the important tag here. We can also conditionally add the default `border-slate-100 dark:border-slate-800`